llmfoundry/data/finetuning/tasks.py

```diff
@@ -118,7 +194,8 @@ def _slice_chat_formatted_example(
         ValueError: If the chat example has less than two messages or if the last message is not from the assistant.
         KeyError: If a message does not have a role or content.
     """
-    messages = example['messages']
+    _validate_chat_formatted_example(example)
```

Comment:
I think there's some validation below that is now unnecessary bc you're doing it here now